CRM/Event/Form/Registration/Confirm.php

```diff
@@ -928,7 +928,6 @@ public function postProcess() {
 
         //send mail to primary as well as additional participants.
         $this->assign('contactID', $contactId);
-        $this->assign('participantID', $participantID);
```

Comment:
@eileenmcnaughton is the participantID field exposed on any confirm step or thank you step form or is this assign just about the smarty email template?

Reply:
yeah after this line the only remaining thing that happens is calling sendEmail + it iterates through multiple participantIDs so it would be the 'last one'